.pre-commit-config.yaml

```diff
@@ -42,7 +42,20 @@ repos:
     rev: v2.1.1
     hooks:
       - id: docker-compose-check
-        exclude: solr6-overrides.yaml
+        files : docker-compose/.*compose.yaml
+
+  - repo: local
+    hooks:
+      - id: check-docker-compose-solr6-overrides
+        name: Check Docker Compose Solr6 Overrides
+        entry: |
+          bash -c "
+          docker compose -f docker-compose/compose.yaml -f docker-compose/solr6-overrides.yaml config |
+          yamllint --config-data '{extends: default, rules: {line-length: disable, document-start: {present: false}}}' -
```

Comment:
what are you overriding here which you can't override in the main yamllint hook?